### DISCUSSION TITLE
pinning new items to start (windows 10 style)

### DISCUSSION THREAD

**Discussion by f2bacon** [valinet/ExplorerPatcher#1439] [Q&A]
i see at the bottom of the start menu section it says "pin tiles to windows 10 start menu from file explorer" and when i click that, it opens file explorer.
but when i click "pin to start" it doesn't pin to the windows 10 style start menu. this is a challenge for programs that dont show up in the start menu all programs section like asus armoury crate or on the stock windows start menu i was able to add a recycle bin shortcut.

**Comment by alexonpeace** [comment]
you can't pin UWP apps to win10 start menu if you're on win11
